### ISSUE TITLE
Draw Control - Undo/Redo options

### ISSUE THREAD

**Issue by MattiasSp** [origo-map/origo#2018]
**Description**
When editing, it would be extremely useful with an undo/redo option. Then we could also do away with the confirmation dialog on deleting.

**Example**
![chrome-capture-2024-5-28-crop](https://github.com/origo-map/origo/assets/2445979/5e9ae229-d026-4bfb-8a29-da4d6a5e5bf2)

**Comment by tonnyandersson** [comment]
I think that an undo/redo function should be able to handle all sorts of edits, not just delete operations. E.g. add/delete/move features/nodes.

I also think that both the editor and the draw control should have this option, but I've been told that the editor may offer some resistance to the implementation of such a feature...

**Comment by MattiasSp** [comment]
I agree, both about handling all modifications (maybe even styling) and about the editor using the same functionality.

The original issue concerned deleting specifically, since that's what causes the slightly annoying confirmation dialog, but it would be surprising in a bad way for a user to learn that it only works for one type of modification.
